pages/index.tsx

```diff
@@ -1,17 +1,33 @@
-import Link from "next/link";
-import dbConnect from "../utils/dbConnect";
+import Link from 'next/link'
+import dbConnect from '../utils/dbConnect'
+import { signIn, signOut, useSession, getSession } from 'next-auth/client'
 
-const Index = () => (
-    <>
-        <h1>That's totally NOT a ToDo List!</h1>
-    </>
-);
+const Index = (props) => {
+    const { session } = props
```

Comment:
Should we really use SSR for retrieving session there instead of `useSession` hook?
According to docs, it seems that `getSession` is mostly used for securing API-routes